evident/effect_size.py

```diff
@@ -0,0 +1,150 @@
+
+# -----------------------------------------------------------------------------
+# Copyright (c) 2018, The Evident Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+# import joblib
+import pickle
+
+import pandas as pd
+import numpy as np
+
+from os.path import join, basename, exists
+
+from itertools import combinations
+from functools import partial
+from skbio import DistanceMatrix
+# from skbio.stats.distance import permanova
+from scipy.stats import mannwhitneyu, spearmanr
+
+
+def effect_size(mappings, alphas, betas, output, jobs, permutations,
+                alpha_method, overwrite, na_values):
+    # As we can have multiple mapping, alpha or files, we will construct a mfs
+    # dictionary with all the dataframes. Additionally, we will load the
+    # data_dictionary.csv file so we can use it to process the data
+    mappings = {f: pd.read_csv(f, sep='\t', dtype=str, na_values=na_values)
```

Comment:
Should we do some kind of validation of the na_values before getting to this point? Is this somewhere else?

Reply:
The current code only checks whether the user defined na_values are strings; but not sure what else can we validate.